packages/site/src/components/Logo/Fox.tsx

```diff
@@ -0,0 +1,433 @@
+/* eslint-disable */
+
+import { vec2, vec3 } from 'gl-matrix';
```

Comment:
Aren't these files available as a package?

Reply:
Also if we are using this can we remove `icon.gif`?

Reply:
> Aren't these files available as a package?
I couldn't find it. In the MetaMask website we do it like this.

> Also if we are using this can we remove `icon.gif`?
Yes.